### ISSUE TITLE
Problems view: filtering does not read out data about items found

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52393]
Environment Details:
VSCode Version : 1.23.1
OS Version : Win10
MAC OS Version: HighSierra10.13.5 

Repro Steps:
1. Launch VS Code while NVDA/VoiceOver is ON.
2. Go to View menu and Select Problems.
3. Tab till Filter textbox.

Actual:
VoiceOver does not read the text whatever we type in the Filter textbox and also does not announce any information about how many results found. Count issue is also observed with NVDA.

Expected:
VoiceOver or any screenreader  should read the text whatever we type in the textbox and also should give information about the data found.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

Attachment for reference: 
[A11y VSCode Problems NVDAVoiceOver not readinmg filter items.pptx](https://github.com/Microsoft/vscode/files/2118052/A11y.VSCode.Problems.NVDAVoiceOver.not.readinmg.filter.items.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by sandy081** [comment]
For me, It reads the letters you type. But agreed it does not read the filtered count.

**Comment by isidorn** [comment]
For me it also reads the letters, but some general information about the filtered count is missing.

**Comment by sandy081** [comment]
I do not think we read the letters we type anywhere in other input boxes. What is important is the results after typing. 

Now it reads the result after typing.
